### ISSUE TITLE
Kernel logs on NetBSD

### ISSUE THREAD

**Issue by jacereda** [jart/cosmopolitan#316]
I'm getting this in `dmesg`:

```
[ 15201.848623] ./demo.com: Unknown elf note type 1 (NetBSD tag): [namesz=8, descsz=4 name=OpenBSD ]
```

Looks harmless, but in case someone faces this situation, it will probably go away in kernels compiled without `options DIAGNOSTIC`:

https://github.com/NetBSD/src/blob/d04203e9b8e767b679649c4f87eada9b1a99ff05/sys/kern/exec_elf.c#L1102

**Comment by jart** [comment]
Which version of NetBSD are you running? I'm running NetBSD 9.1. It's possible that they changed something since then. The NetBSD note is placed after the OpenBSD note in the ape/ape.S bootloader. I was so happy when I discovered that worked, since the dimensions of notes sometimes differs across platforms.

Try troubleshooting things by changing the order of notes. My fear is that we might find ourselves in a situation where NetBSD support has become mutually exclusive from OpenBSD support. In that case, we'll need to upstream a change with the NetBSD team. I've already established a relationship with them since I helped get an issue fixed with their /bin/sh earlier. Then in the meantime we can simply recommend people run NetBSD 9.1.
